foerder.j2

```diff
@@ -0,0 +1,266 @@
+<html>
```

Comment:
Please git mv the foerder.html to foerder.j2 in the relevant commit so we don't loose the history of this file.

Reply:
Hm both quite true. It would be good to keep a copy to we can serve it directly via github pages.. If we'll keep both, we don't really have to move it. 👍